src/js/Controllers/DisplayCapabilities.js

```diff
@@ -686,7 +710,9 @@ let capabilities = {
 			"screenParams": screenParams,
 			"imageCapabilities": ["DYNAMIC", "STATIC"],
 			"menuLayoutsAvailable": ["LIST", "TILES"]
-		}
+		},
+		"softButtonCapabilities": [],
```

Comment:
```suggestion
```
Seems to be an issue from merge conflicts, this should not be here any more

Reply:
@jacobkeeler please find fix [here](https://github.com/LuxoftSDL/generic_hmi/pull/41/commits/608df7fe0daf02466cc119bfc85bf870e5673610)